scene/gui/file_dialog.cpp

```diff
@@ -756,7 +754,7 @@ void FileDialog::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_unhandled_input"), &FileDialog::_unhandled_input);
 
 	ClassDB::bind_method(D_METHOD("_tree_selected"), &FileDialog::_tree_selected);
-	ClassDB::bind_method(D_METHOD("_tree_db_selected"), &FileDialog::_tree_dc_selected);
+	ClassDB::bind_method(D_METHOD("_tree_item_activated"), &FileDialog::_tree_item_activated);
```

Comment:
Actually my bad, the original name needs to be kept for the binding to avoid breaking compatibility (but the C++ name can still be changed). Will add this one to #16863.

Reply:
Given that it starts with an underscore, I think this is not a problem.

This won't be listed in documentation or autocompleted in GDScript. If anyone uses underscore-starting methods, they should be prepared to fix their messy hacks.

Reply:
Fair enough. :)